compiler-utils/build.gradle

```diff
@@ -1,10 +1,13 @@
 apply plugin: 'org.jetbrains.kotlin.jvm'
+apply plugin: 'java-test-fixtures'
```

Comment:
I'm using test fixtures for testing utilities. That seemed to be a good use case here.